### ISSUE TITLE
[Accessibility]WPF CheckBox is read twice by NVDA scan mode

### ISSUE THREAD

**Issue by wangqiaodong** [dotnet/wpf#5232]
* .NET Core Version: 
* Windows version: windows 10 20H2
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? If yes, please file the issue via the instructions [here](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2019).No
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs). No

 
 
 **Problem description:**

Create a WPF window with following code: 
`<StackPanel>
      <CheckBox Content="option 1"
                IsChecked="True"
                HorizontalAlignment="Center"
                VerticalAlignment="Center" />
</StackPanel>`

then open NVDA tool (https://www.nvaccess.org/download/) and use scan mode to read the window, you will get following read result:

MainWindow  window
Object nav mode
Web mode
Scan mode
System  collapsed  Space
option 1  check box  checked
option 1

 **Actual behavior:** 

Read the checkbox need to press keyboard down arrow twice, first time will read  "option 1  check box  checked" and second time will read "option 1".

 **Expected behavior:**
 For the checkbox, we only need one step down arrow to navigate it.

 **Minimal repro:**

**Comment by anjalisheel-wpf** [comment]
@wangqiaodong This is working according to the design. When you will put focus on the checkbox then it will read - "option 1 check box unchecked (or checked - based upon the state)".

But if you will toggle its state then narrator(or NVDA) is going to announce toggle state i.e. checked and unchecked.

If you will try this same thing with radio button then also you will notice the same behavior.
